components/sup/src/manager/commands.rs

```diff
@@ -205,7 +205,12 @@ pub fn service_load(mgr: &ManagerState,
             // desired package identifier, it will be used;
             // otherwise, we'll install the latest suitable
             // version from the specified Builder channel.
-            util::pkg::satisfy_or_install(req, &source, &bldr_url, &bldr_channel)?;
+            let package = util::pkg::satisfy_or_install(req, &source, &bldr_url, &bldr_channel)?;
+            if let Err(e) = spec.validate(&package) {
```

Comment:
In retrospect, it's very odd that this _wasn't_ here before... weird.

Good catch!